src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,63 @@
 
 import org.springframework.stereotype.Service;
 
+/**
+ * Service class providing basic calculator operations.
+ */
```

Comment:
6. **Method naming**: The method names are clear and concise, but it might be a good idea to provide more detailed method names for better readability and understanding.